src/chapters/razor-views/template-forms.rst

```diff
@@ -0,0 +1,114 @@
+Razor Forms
+===========
+
+Already we have seen that templates allow us to build generic forms.
+Using templates, you can reuse the structure by rendering the same form, but with different labels and data.
+Thus, a single form can serve different purposes, saving you extra effort.
+
+Whenever possible, reuse existing templates!
+
+Start a New Project
+-------------------
+
+You will build a new project so you can practice with templates and forms.
+If you have not done so, commit and push any unsaved work from your ``HelloASPDotNET`` project.
+
+.. TODO: Add link back to setup steps
+
+Your new project will keep track of some fictional coding events.
+To get started, follow the steps you took to create ``HelloASPDotNET``, but call the project ``CodingEvents``.
```

Comment:
Is there a previous page or video that we can link to here?

Reply:
Done!